docs/docker/build.sh

```diff
@@ -2,13 +2,13 @@
 set -e
 set +x
 
-if [[ ($1 == '--help') || ($1 == '-h') ]]; then
-  echo "usage: $(basename $0)"
+if [[ ($1 == '--help') || ($1 == '-h') || ($1 == '') ]]; then
+  echo "usage: $(basename $0) {bionic,focal}"
   echo
-  echo "Build Playwright docker image and tag it as 'playwright:localbuild'."
+  echo "Build Playwright docker image and tag it as 'playwright:localbuild-{bionic,focal}'."
   echo "Once image is built, you can run it with"
   echo ""
-  echo "  docker run --rm -it playwright:localbuild /bin/bash"
+  echo "  docker run --rm -it playwright:localbuild-{bionic,focal} /bin/bash"
```

Comment:
`docs/docker/tag_and_push.sh` should be updated too according to these changes.

Let's accept image name as an argument both here and in `docs/docker/tag_and_push.sh`?